### DISCUSSION TITLE
Integrate PuLID on forge

### DISCUSSION THREAD

**Discussion by evelryu** [lllyasviel/stable-diffusion-webui-forge#1881] [Ideas]
This would be a great feature for Forge. 
https://github.com/ToTheBeginning/PuLID/blob/main/docs/pulid_for_flux.md

**Comment by a3nima** [comment]
I'm not sure, maybe the controlnet people could make it work in controlnet?

**Reply by evelryu** [reply]
This would be amazing.

**Reply by a3nima** [reply]
but I dont think so, they havent even added FLUX support, as far as I've seen it ._.

**Comment by kaliumlive** [comment]
why is no one talking about this?

**Comment by wzwow** [comment]
can put this in forge space?? forge space was a great idea but nobody updating it

**Comment by CaptDagon** [comment]
this would be great to have, how hard it can be to implement?
